nbconvert/preprocessors/csshtmlheader.py

```diff
@@ -7,11 +7,19 @@
 import os
 import io
 import hashlib
+import nbconvert.resources
 
 from traitlets import Unicode
 from ipython_genutils.py3compat import str_to_bytes
 from .base import Preprocessor
 
+
+try:
+    from notebook import DEFAULT_STATIC_FILES_PATH
```

Comment:
Doesn't the try-except fallback specifically make it *not* dependency for all of nbconvert?